integration-tests/test-oncotree-dependent-tools.sh

```diff
@@ -125,6 +127,19 @@ if [ $ONCOTREE_DEPLOYMENT_SUCCESS -gt 0 ] ; then
     else
         ONCOTREE_CODE_CONVERTER_OUTPUT_TEST_SUCCESS=1
     fi
+
+    # this test is data dependent (will fail if versions are changed in topbraid history) 
+    EXPECTED_ONCOTREE_VERSION_MAPPER_OUTPUT=$INTEGRATION_TEST_DIRECTORY/default_oncotree_file_converted.txt
+    TEST_ONCOTREE_VERSION_MAPPER_INPUT_FILENAME=$TESTING_DIRECTORY_TEMP/test_oncotree_mapper_version.txt
+    TEST_ONCOTREE_VERSION_MAPPER_OUTPUT_FILENAME=$TESTING_DIRECTORY_TEMP/test_oncotree_mapper_version_output.txt
+    cp $MOCK_ONCOTREE_FILE $TEST_ONCOTREE_VERSION_MAPPER_FILENAME
+    python $ONCOTREE_SCRIPTS_DIRECTORY/cross_version_oncotree_translator.py -u "$ONCOTREE_URL/api/" -s oncotree_2019_03_01 -t oncotree_2018_05_01 -i $TEST_ONCOTREE_VERSION_MAPPER_INPUT_FILENAME -o $TEST_ONCOTREE_VERSION_MAPPER_OUTPUT_FILENAME
+    diff $EXPECTED_ONCOTREE_VERSION_MAPPER_OUTPUT $TEST_ONCOTREE_VERSION_MAPPER_OUTPUT_FILENAME
+    if [ $? -gt 0 ] ; then
+        echo "cross_version_oncotree_translator.py output differs from expected output"
+    else
+        ONCOTREE_VERSION_MAPPER_TEST_SUCCESS=1
+    fi
 fi
 
 rm -rf $TESTING_DIRECTORY_TEMP
```

Comment:
Should there be some sort of sanity check here to make sure this directory exists and it does not equal `/`? I recall @mandawilson and Zack accidentally cleaning up a root directory on dashidev I believe because of something like this.